apps/src/templates/sectionProgressV2/SkeletonProgressDataColumn.jsx

```diff
@@ -33,6 +32,6 @@ export default function SkeletonProgressDataColumn({lesson, sortedStudents}) {
 }
 
 SkeletonProgressDataColumn.propTypes = {
-  sortedStudents: PropTypes.arrayOf(studentShape),
+  sortedStudents: PropTypes.array,
```

Comment:
Was the `studentShape` an inaccurate representation of the data here?

Reply:
Yes, the skeleton data only only has IDs until the students are loaded.